src/app/pages/i18n/i18n.component.html

```diff
@@ -0,0 +1,100 @@
+<mat-sidenav-container class="i18n-container" (backdropClick)="open = false">
+    <mat-sidenav mode="over" class="sidenav" [opened]="open" position="end">
+        <mat-nav-list fxLayout="column" fxFlexFill class="sidenav-content">
+            <mat-toolbar class="toolbar-header">
+                <mat-toolbar-row class="logo-row">
+                    <mat-icon class="logo">offline_bolt</mat-icon>
+                </mat-toolbar-row>
+                <mat-toolbar-row style="height: 4px;">
+                    <small>PX {{ 'BLUE' | translate }}</small>
+                </mat-toolbar-row>
+                <mat-toolbar-row>
+                    <small style="margin-top: -24px;" id="i18n">{{ 'I18N' | translate }}</small>
+                </mat-toolbar-row>
+            </mat-toolbar>
+            <mat-divider></mat-divider>
+            <div class="nav-wrapper">
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>home</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HOME' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>folder</mat-icon>
+                    <span>{{ 'MENU_ITEMS.PROJECTS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>report</mat-icon>
+                    <span>{{ 'MENU_ITEMS.WARNINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>settings</mat-icon>
+                    <span>{{ 'MENU_ITEMS.SETTINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>help</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HELP' | translate }}</span>
+                </a>
+            </div>
+        </mat-nav-list>
+    </mat-sidenav>
+    <mat-sidenav-content class="content">
+        <mat-toolbar color="primary">
+            <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+                <mat-icon>menu</mat-icon>
+            </button>
+            <div>{{ 'I18N' | translate }}</div>
+            <pxb-spacer></pxb-spacer>
+            <button
+                type="button"
+                aria-label="Toggle sidenav"
+                mat-icon-button
+                (click)="openI18nMenu()"
+                style="margin-left: -4px; margin-right: 12px;"
+            >
+                <mat-icon aria-label="Side nav toggle icon">menu_open</mat-icon>
```

Comment:
We should flip this icon (scaleX -1) when we're viewing RTL languages

Reply:
Added this in so the shopping cart and secondary menu icon are inverted for RTL languages now.